### PR TITLE
Fix invalid null delegate usage attempts in media clusters

### PR DESCRIPTION
**Pull request by lazarkov**
#### Problem
@andreilitvin  pointed out that in media cluster code quote: _"Delegate null check was not valid: even if null check failed, delegate use was still attempted."_

#### Change overview
see above

#### Issue(s) fixed:
[20683](https://github.com/project-chip/connectedhomeip/issues/20683)

#### Testing
Run `./scripts/run_in_build_env.sh \  
                  "./scripts/tests/run_test_suite.py \
                     --chip-tool ./out/debug/standalone/chip-tool \
                     run \
                     --iterations 1 \
                     --all-clusters-app ./out/debug/standalone/chip-all-clusters-app \
                     --tv-app ./out/debug/standalone/chip-tv-app \
                  "`

### CODE REVIEW

**Review by chrisdecenzo — APPROVED** [review]

File: src/app/clusters/account-login-server/account-login-server.cpp [hunk 1]
```diff
@@ -136,41 +136,50 @@ bool emberAfAccountLoginClusterLoginCallback(app::CommandHandler * command, cons
...
+    if (!delegate->HandleLogin(tempAccountIdentifier, setupPin))
+    {
+        status = EMBER_ZCL_STATUS_NOT_AUTHORIZED;
+    }
+
 exit:
     if (err != CHIP_NO_ERROR)
     {
         ChipLogError(Zcl, "emberAfAccountLoginClusterLoginCallback error: %s", err.AsString());
         emberAfSendImmediateDefaultResponse(EMBER_ZCL_STATUS_FAILURE);
```
Comment (andy31415) [inline src/app/clusters/account-login-server/account-login-server.cpp:155]: if err is not CHIP_NO_ERROR this seems to send emberAfSendImmediateDefault report twice. Is this intentional? I would assume no.

Comment (andy31415) [reply src/app/clusters/account-login-server/account-login-server.cpp:155]: Same comment for methods below.

**Review by andy31415 — COMMENTED** [review]
@lazarkov - changes in feature map should be independent of cpp error handling logic. Separate PRs for each would make more sense.

**Comment by lazarkov** [comment]
> @lazarkov - changes in feature map should be independent of cpp error handling logic. Separate PRs for each would make more sense.

It came as an additonal request from @chrisdecenzo. I believe it's per spec to have feature map of network commissioning set to 4. @chrisdecenzo can confirm.

I understand the logic to move this change to new PR, in case we want to roll the change back. But per my understanding, this should be the default value.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**